packages/url_launcher/url_launcher_platform_interface/lib/method_channel_url_launcher.dart

```diff
@@ -27,15 +27,14 @@ class MethodChannelUrlLauncher extends UrlLauncherPlatform {
   }
 
   @override
-  Future<bool> launch(
-    String url, {
-    @required bool useSafariVC,
-    @required bool useWebView,
-    @required bool enableJavaScript,
-    @required bool enableDomStorage,
-    @required bool universalLinksOnly,
-    @required Map<String, String> headers,
-  }) {
+  Future<bool> launch(String url,
+      {@required bool useSafariVC,
+      @required bool useWebView,
+      @required bool enableJavaScript,
+      @required bool enableDomStorage,
+      @required bool universalLinksOnly,
+      @required Map<String, String> headers,
+      String webOnlyLinkTarget}) {
```

Comment:
Let's call the parameter:

`webOnlyWindowName` everywhere.

Also: add a comma after the last named parameter, so the `dartfmt` looks more like the previous version :)